components/network/quagga/patches/14-cve-2013-2236.patch

```diff
@@ -0,0 +1,114 @@
+This patch may be removed once Quagga is updated to 0.99.22.2 or
```

Comment:
Can we try import new quagga?

Reply:
I would like to, but unfortunately I do not have the time to test if the newer source will build AND work correctly, so I'd like to stick to the solaris-userland version.